data/net/minecraft/util/Mth.mapping

```diff
@@ -199,6 +252,40 @@ CLASS net/minecraft/util/Mth
 	METHOD positiveModulo (II)I
 		ARG 0 x
 		ARG 1 y
+	METHOD quantize (DI)I
+		COMMENT Gets the value closest to zero that is not closer to zero than the given value and is a multiple of factor.
+		ARG 0 value
+		ARG 2 factor
+	METHOD randomBetween (Ljava/util/Random;FF)F
+		ARG 0 random
+		ARG 1 min
+		ARG 2 max
+	METHOD randomBetweenInclusive (Ljava/util/Random;II)I
+		ARG 0 random
+		ARG 1 minInclusive
+		ARG 2 maxInclusive
+	METHOD rayIntersectsAABB (Lnet/minecraft/world/phys/Vec3;Lnet/minecraft/world/phys/Vec3;Lnet/minecraft/world/phys/AABB;)Z
+		ARG 0 start
+		ARG 1 end
+		ARG 2 boundingBox
+	METHOD rotLerp (FFF)F
+		COMMENT Method for linear interpolation of rotations
```

Comment:
Ideally, this should be more of a sentence describing what it does rather than what it is. ("Linearly interpolates between the start and end values as degrees in rotation." or something similar, perhaps?)